client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoDatabaseMetaData.java

```diff
@@ -1433,13 +1433,11 @@ public ResultSet getClientInfoProperties()
 
         Stream.of(ClientInfoProperty.values())
                 .sorted(Comparator.comparing(ClientInfoProperty::getPropertyName))
-                .forEach(clientInfoProperty -> {
-                    results.add(newArrayList(
-                            clientInfoProperty.getPropertyName(),
-                            VARCHAR_UNBOUNDED_LENGTH,
-                            null,
-                            null));
-                });
```

Comment:
i think at times i would prefer the former. 
not every use of braces `() -> { single-line; }` is a bad one.